### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by Murkur** [midspace/SEToolbox#127]
not sure how to make a suggestion but the ability to generate blueprint files that can be loaded with the blueprint spawner would be cool, generating prefabs dont work and just crashes the game

**Comment by midspace** [comment]
An in export, directly to your blueprints folder?
`%appdata%\SpaceEngineers\Blueprints\local`

**Comment by Murkur** [comment]
yes ive tried exporting a sbc prefab but attempting to load it causes a large memory leak and then a game crash it may be the new update released a few days ago

**Comment by Murkur** [comment]
for it to create a blueprint file with the correct names along with the sbcBP file

**Comment by midspace** [comment]
The ability to export to blueprints has now been released:
https://github.com/midspace/SEToolbox/releases/latest

Though I may look at tweaking a few things before closing this.
Specifically:
- [ ] Overwriting should use existing blueprint details (DisplayName, WorkshopId, OwnerSteamId) instead of wiping them.
- [ ] There isn't a thumbnail either. We obviously can't generate one of the ship, but at least should have a nice SEToolbox png there instead.
